### ISSUE TITLE
!is:highest doesn't work

### ISSUE THREAD

**Issue by jondthompson** [dcaslin/d2-checklist#502]
It returns all weapons instead of omitting the highest one.

**Comment by dcaslin** [comment]
Hmmm I just tested on mine and it works properly. 4 guns are my highest, !is:highest returns 393, and no filter returns 397.

![image](https://user-images.githubusercontent.com/16437968/159540510-48edac2c-69c4-485c-984a-75a5d06b2cef.png)

![image](https://user-images.githubusercontent.com/16437968/159540571-75f60007-e462-48aa-abce-d09ec9d8ccc8.png)

![image](https://user-images.githubusercontent.com/16437968/159540622-664da99d-ee14-4ddd-986f-e08db9efbdb8.png)

**Comment by jondthompson** [comment]
I think I understand my confusion.  is:highest is looking at the highest power weapons of all weapons. I was thinking of it as the highest power version of each weapon.  

So if we use as an example three Razor's Edges (1557, 1550, and 1350), two Sweet Sorrows (1540, and 1520), and a single Xenophage that is at 1350.  
I believe now that the current code for is:highest will only show one 1557 razor's edge. So !is:highest shows all but that 1557. 

I was looking at it as is:highest should be showing one 1557 Razor's Edge, one 1540 Sweet Sorrow, and one 1350 Xenophage, as those are the highest copy of each gun. So !is:highest would show me two razor's edges (1550 and 1350) and one sweet sorrow (1520), which if they are also "Not good/god/deepsight/crafted" I can bulk tag them to be deleted.

Basically I'm trying to identify via search which untagged weapons can be deleted, and those that are the highest light level copy of the same gun should never be deleted. They should either be infused or replace the version I have marked as 'upgrade'.

**Comment by dcaslin** [comment]
So I tend to think about guns more by frame/slot/energy than by individual guns. Which is why I have the "Compare" and the shortcut for tags by Slot. In the screenshot below I have 6 Lightweight Kinetic shotguns (too many!) 

![image](https://user-images.githubusercontent.com/16437968/159574082-329c7a60-9afa-40ad-ac01-95455ebc9f31.png)

![image](https://user-images.githubusercontent.com/16437968/159574249-f11c4ed0-cd9d-454c-a917-6c0f0429392b.png)

I tend to use that compare window to figure it out. 

Alternatively I also like to sort by "Copies" to find guns where I have too many

![image](https://user-images.githubusercontent.com/16437968/159574351-80d86aa9-6552-47d2-9880-f2f41d8bbdaf.png)

**Comment by jondthompson** [comment]
Yeah, I've been using the copies screen to filter through them. I was just thinking that I could better bulk filter all of the ones I know I'll shard via "untagged AND not highest AND Not good/god/deepsight/crafted".
